commit 568552e4fd1fbec7e38fb6876c5debfb1a24ae2f
Author: Christian Muehlhaeuser <muesli@gmail.com>
Date:   Fri Sep 16 11:19:22 2011 +0300

    Should be qt3 instead of qt4 in the description and summary - I'd guess :-)

diff --git a/klfbackend-qt3.manifest b/klfbackend-qt3.manifest
index 8d330aa..eeffbc5 100644
--- a/klfbackend-qt3.manifest
+++ b/klfbackend-qt3.manifest
@@ -2,10 +2,10 @@
   "schema_version": 1,
   "name": "klfbackend-qt3",
   "version": "3.2.2",
-  "summary": "KLatexFormula backend library (Qt4)",
+  "summary": "KLatexFormula backend library (Qt3)",
   "homepage": "http://klatexformula.sourceforge.net/",
   "licenses": ["GPLv2+"],
-  "description": "A C++/Qt library to generate images (PNG, EPS, PDF, plus all Qt-supported image\nformats) from LaTeX equations.\n\nThis library implements the base functionality of KLatexFormula. This library\nis compiled for Qt 4.\n\nSee also packages klatexformula and libklfbackend-qt3.\n",
+  "description": "A C++/Qt library to generate images (PNG, EPS, PDF, plus all Qt-supported image\nformats) from LaTeX equations.\n\nThis library implements the base functionality of KLatexFormula. This library\nis compiled for Qt 3.\n\nSee also packages klatexformula.\n",
   "authors": [],
   "maturity": "stable",
   "packages": {
